tests/test_hierarchy_helpers.hpp

```diff
@@ -50,11 +51,30 @@ class Source final : public dealii::Function<dim>
 };
 
 template <int dim>
-class ConstantMaterialProperty final : public dealii::Function<dim>
+class Coefficient : public dealii::Function<dim>
+{
+public:
+  virtual dealii::VectorizedArray<double>
+  value(dealii::Point<dim, dealii::VectorizedArray<double>> const &p,
+        unsigned int const = 0) const = 0;
```

Comment:
Where is this function actually used?

Reply:
https://github.com/ORNL-CEES/mfmg/blob/master/tests/laplace_matrix_free.hpp#L109

Reply:
Thanks!